riva/proto/riva_nmt.proto

```diff
@@ -75,6 +75,7 @@ message TranslationConfig {
   string source_language_code = 1;
   string target_language_code = 2;
   string model_name = 3;
+  repeated string dnt_phrases = 4;
```

Comment:
will need to add a description of this field here

Reply:
Its done

Reply:
will be good to add a more descriptive comment like here https://github.com/nvidia-riva/common/blob/main/riva/proto/riva_asr.proto#L269